### ISSUE TITLE
Free ID that is showing unclaimed in Explorer is showing as unavailable in the Browser

### ISSUE THREAD

**Issue by ryanarndtcm** [blockstack/blockstack-browser#2059]
This is coming in from a user in support who has been sent a link to this issue to add to. I will add anything else in here that they send to support.

shows ID is not taken: https://explorer.blockstack.org/name/andonov.id.blockstack

Screenshot.png
![image](https://user-images.githubusercontent.com/52256509/80751163-b3705b00-8aff-11ea-99c8-882abc446645.png)

Shows ID is taken

**Comment by ryanarndtcm** [comment]
"I tried on Firefox, Brave, Iridium - get the same thing"

**Comment by timstackblock** [comment]
Yes this isn't browser based it seems to be a data issue with name registration. Let me ping Hank and see if he has noticed this issue. @hstove

**Comment by ryanarndtcm** [comment]
@timstackblock user is checking in on the status of this, is there anything I can update them with?
thanks!

**Comment by timstackblock** [comment]
I spoke to @hstove about this and he found the issue, I will assign this ticket to him. 

What we found it that we just need better validation on this form. The issue is no the number of characters users are trying to register names with, thats what causes the error.

**Comment by hstove** [comment]
To clarify the issue here: subdomains must be **at least** 8 characters. This one is 7, which is why it's being blocked from registering.

**Comment by ryanarndtcm** [comment]
Ahh, this is a carry over ID from Onename which was registered and is now expired and cannot be renewed, so for this person, it will then be lost from the sounds of it. 

Is that correct?
